activities/GameOfLife.activity/js/activity.js

```diff
@@ -32,7 +32,7 @@ function main(Board, State, patterns, color, shadeColor, l10n, dataStore) {
 
   var storeLocally = function storeLocally(state) {
     dataStore.setDataAsText({
-      state: state
+      state: state,
```

Comment:
I guess you planned to add generation here ?

Reply:
Yes I'll make this fix.